src/components/Row/Row.js

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'react-emotion';
+
+import { clearfix } from '../../styles/style-helpers';
+
+const baseStyles = ({ theme }) => css`
```

Comment:
Is there a reason we can't just position the `Grid` component itself relatively, if necessary via a boolean prop? I thought about having a `Row`, but decided against it eventually to reduce the number of components someone would need for such simple layouts. I know you only need it if you want to use `skip`, but that might also add a bit of ambiguity. What do you think?